keyboards/cannonkeys/satisfaction75/prototype/prototype.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "satisfaction75.h"
+
+#define LAYOUT_default( \
+	K000,       K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013,             \
```

Comment:
I hate you for doing Kxyz, instead of K(row)(col), but I like the board so I'll let it pass.

Reply:
It is K(row)(col)! I just didn't use hex numbers :P
Only 1 digit for row, 2 for col